tests/test_errors.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Flask--Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Errors test module."""
+
+import pytest
+from flask import Flask
+
+from flask_resources.resources import CollectionResource, Resource, ResourceConfig
+
+
+# These classes are in the file because they are under test too
+class CustomConfig(ResourceConfig):
+    """Custom resource configuration."""
+
+    item_route = "/custom/<id>"
+    list_route = "/custom/"
+
+
+class CustomResource(CollectionResource):
+    """Custom resource implementation."""
+
+    def __init__(self, *args, **kwargs):
+        """Constructor."""
+        super(CustomResource, self).__init__(config=CustomConfig, *args, **kwargs)
+
+
+@pytest.fixture(scope="module")
```

Comment:
moderate: Move to conftest.py?

Reply:
I had a comment about this: https://github.com/inveniosoftware/flask-resources/pull/53/files#diff-68876137696247abc8c123622c73a11fR16 

Here is my take: Resource and ResourceConfig are actually under test here. Especially for the follow-up work we need to configure them differently and see if the configuration works. We don't want to break other tests though, so we isolate app creation in the file.